src/config/twitterAuth.js

```diff
@@ -0,0 +1,10 @@
+import SITE_URL from "./constants"
+
+export const configTwitter = {
+    'twitterAuth' : {
+        'consumerKey'       : 'gpxS6tBGRIw9biXdMT9oMVXxB',
+        'consumerSecret'    : 'RSkfOidDZUKIOJClFPBC1yY0W4kDEYQx7ASYoalYPRjAszPxKU ',
+        'callbackURL'       : `${SITE_URL}/auth/twitter/callback`
```

Comment:
these should be inside `.env` and then share with the team. We cannot expose these credentials to the public.

Reply:
Alright... done